packages/client/src/utils/isEmpty.ts

```diff
@@ -0,0 +1,15 @@
+export const isEmpty = (value: unknown) => {
```

Comment:
Рекомендую выработать привычку добавлять возвращаемое значение функции, хотя бы для таких вещей как utils, запросы к апи (я бы даже сказал must have тут), это хорошая практика:
```suggestion
export const isEmpty = (value: unknown): boolean => {
```